setup.cfg

```diff
@@ -26,7 +26,7 @@ install_requires =
     psutil>=5.7.2
     numpy>=1.16
     astropy>=4.0
-    romanad==0.3.0
+    romanad>=0.3.0
```

Comment:
Can you change this temporarily to the following so that we can see the tests pass?

```
romanad @ https://github.com/PaulHuwe/rad.git@RAD-24_DQviaSTCAL
```